index.js

```diff
@@ -11,14 +12,20 @@ module.exports = (robot) => {
       'pull_request.unlabeled',
       'pull_request.synchronize'
     ],
-    (context) => { Handler.handlePullRequest(context) }
+    context => Handler.handlePullRequest(context)
   )
 
   robot.on(['issues.milestoned', 'issues.demilestoned'],
-    (context) => { Handler.handleIssues(context) }
-  )
-
-  robot.on(['issues.opened'],
-    (context) => { Handler.handleIssuesOpened(context) }
+    context => Handler.handleIssues(context)
+  )
+
+  robot.on(['issues.opened'],
+    (context) => { Handler.handleIssuesOpened(context) }
+  )
+
+  // check every two seconds.
+  scheduler(robot, { interval: 60 * 60 * 2 })

```

Comment:
i am not sure how scheduler works but from the comment it seems like we check for staleness every two seconds is that correct? if so is there any check to make sure we are not spamming `createComment` every two seconds?

Reply:
`createComment` will cause the issue/pull to have an updated timestamp and hence the search query will not return that issue/pull again till there's no activity since `n` days.